Makefile

```diff
@@ -123,12 +106,38 @@ load-db: login
 	docker-compose -f docker-compose.yml -f docker-compose.load-db.yml run load-db-dataset
 	docker-compose -f docker-compose.yml -f docker-compose.load-db.yml run load-db-entity
 
-cf-login:
-	cf target -o dluhc-digital-land || cf login -a api.london.cloud.service.gov.uk
+cf-check:
+# install dependencies
+ifndef CF_CLI
+ifeq ($(UNAME),Darwin)
+$(error CloudFoundry CLI not found in PATH)
+endif
+	curl https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+	echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+	sudo apt update
+	sudo apt install -y cf8-cli
+endif
+
+
+cf-login: cf-check
+	cf api https://api.london.cloud.service.gov.uk
+	cf auth
 
 cf-deploy: cf-login
 ifeq (, $(ENVIRONMENT))
 	$(error "No environment specified via $$ENVIRONMENT, please pass as make argument")
 endif
 	cf target -o dluhc-digital-land -s $(ENVIRONMENT)
 	cf push $(ENVIRONMENT)-$(CF_BASE_APP_NAME)
```

Comment:
There's an existing mechanism for running migrations on digital-land.info via `prestart.sh` which is registered as a hook via the `PRE_START_PATH` env var (see https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/start.sh#L23)

Reply:
Ok, does this also have a mechanism in place to avoid race conditions with multiple instances starting? Or shall we tackle that in another PR / Ticket?